dbt/include/fabric/macros/materializations/models/table/columns_spec_ddl.sql

```diff
@@ -1,3 +1,10 @@
+{% macro build_columns_constraints(relation) %}
+    {{ return(adapter.dispatch('build_columns_constraints', 'dbt')(relation)) }}
+{% endmacro %}
+
+{%- macro default__build_columns_constraints(relation) -%}
```

Comment:
Recommend:
- explicit `{{ return('') }}`
- if this is here because it isn't supported on `dbt-synapse`, then put this logic in `dbt-synapse` instead